packages/core/saved-objects/core-saved-objects-base-server-internal/src/serialization/serializer.ts

```diff
@@ -144,11 +146,13 @@ export class SavedObjectsSerializer implements ISavedObjectsSerializer {
       references,
       coreMigrationVersion,
       typeMigrationVersion,
+      managed,
     } = savedObj;
     const source = {
       [type]: attributes,
       type,
       references,
+      ...(managed != null ? { managed } : {}),
```

Comment:
Same question, wouldn't be backfilling with the default safer here?

Reply:
Same [answer](https://github.com/elastic/kibana/pull/154515#discussion_r1162584999)